src/QMCHamiltonians/ECPComponentBuilder.2.cpp

```diff
@@ -100,6 +107,21 @@ void ECPComponentBuilder::buildSemiLocalAndLocal(std::vector<xmlNodePtr>& semiPt
       vpsPtr.push_back(cur_vps);
       Lmax = std::max(Lmax, l); //count the maximum L
     }
+    else if (vname== "vps_so") //This accumulates the spin-orbit corrections, if defined.
+    {
+      app_log()<<"Yo.  We got a vps-so up in heah\n";
```

Comment:
Comment from development should be removed.

Reply:
Wooooops.  Good catch!